src/sidebar/components/excerpt.js

```diff
@@ -12,24 +12,20 @@ import { applyTheme } from '../util/theme';
  * it is expanded or collapsed.
  */
 function InlineControls({ isCollapsed, setCollapsed, linkStyle = {} }) {
-  const toggleTitle = isCollapsed
-    ? 'Show the full excerpt'
-    : 'Show the first few lines only';
   const toggleLabel = isCollapsed ? 'More' : 'Less';
 
   return (
     <div className="excerpt__inline-controls">
       <span className="excerpt__toggle-link">
-        {/* FIXME-A11Y */}
-        {/* eslint-disable-next-line jsx-a11y/anchor-is-valid */}
-        <a
-          href="#"
+        <button
+          className="excerpt__toggle-button"
           onClick={() => setCollapsed(!isCollapsed)}
-          title={toggleTitle}
+          aria-label="Toggle to show the full excerpt or first few lines only"
```

Comment:
Experimenting a bit with the screen reader, i found it more clear when the label didn't change and the aria-pressed state did. Alternatively we could go back to a changing title/aria-label but I would recommend we remove aria-pressed entirely. I think this way is more standard.

Reply:
Because we change the text (and thus the "context") of the button when it is pressed, I do not believe it "counts" as an ARIA toggle button. That is, it is never "pressed" because the title/labeling/whatever changes when the state changes. Does that make sense?